GenumKit/Stencil/Contexts.swift

```diff
@@ -151,7 +151,8 @@ extension StringsFileParser {
           "count": entry.types.count,
           "types": entry.types.map { $0.rawValue },
           "declarations": (0..<entry.types.count).map { "let p\($0)" },
-          "names": (0..<entry.types.count).map { "p\($0)" }
+          "names": (0..<entry.types.count).map { "p\($0)" },
```

Comment:
Code style: I just realized that with Swift 2 now we can use `entry.types.indices` rather than `0..<entry.types.count` :wink: